ruby_connectors/spec/duckdb_connector/connection_spec.rb

```
@@ -16,9 +16,9 @@
     }
   end
 
-  describe "#connect", :vcr do
+  describe "#connect" do
     it "establishes a connection and performs a simple query" do
-      VCR.use_cassette("motherduck_connect") do
+      VCR.use_cassette("duckdb/motherduck_connect") do
         expect { subject.connect }.not_to raise_error
```

rubocop

[convention] Name your test subject if you need to reference it explicitly.

(RSpec/NamedSubject)